app/models/views/processed_data.rb

```diff
@@ -0,0 +1,80 @@
+# coding: utf-8
+module Views
+  class ProcessedData
+
+    def initialize(application)
+      @application = application
+    end
+
+    def application_processed
+      build_return_hash @application
+    end
+
+    def application_deleted
```

Comment:
I would follow the same pattern here as the other public methods - to be one liners which delegate the hash creation to a private method.